azurerm/internal/services/network/client/client.go

```diff
@@ -45,6 +45,8 @@ type Client struct {
 	WebApplicationFirewallPoliciesClient *network.WebApplicationFirewallPoliciesClient
 	PrivateDnsZoneGroupClient            *network.PrivateDNSZoneGroupsClient
 	PrivateLinkServiceClient             *network.PrivateLinkServicesClient
+	ServiceAssociationLinksClient        *network.ServiceAssociationLinksClient
+	ResourceNavigationLinksClient        *network.ResourceNavigationLinksClient
```

Comment:
Can we make these singular rather than plural?

```suggestion
	ServiceAssociationLinkClient        *network.ServiceAssociationLinksClient
	ResourceNavigationLinkClient        *network.ResourceNavigationLinksClient
```

Reply:
Thanks, will change it.